go.mod

```diff
@@ -1,250 +1,257 @@
-	k8s.io/code-generator => k8s.io/code-generator v0.25.11
-	k8s.io/component-base => k8s.io/component-base v0.25.11
-	k8s.io/component-helpers => k8s.io/component-helpers v0.25.11
-	k8s.io/controller-manager => k8s.io/controller-manager v0.25.11
-	k8s.io/cri-api => k8s.io/cri-api v0.25.11
-	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.25.11
-	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.25.11
-	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.25.11
-	k8s.io/kube-proxy => k8s.io/kube-proxy v0.25.11
-	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.25.11
-	k8s.io/kubectl => k8s.io/kubectl v0.25.11
-	k8s.io/kubelet => k8s.io/kubelet v0.25.11
-	k8s.io/kubernetes => k8s.io/kubernetes v1.25.11
-	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.25.11
-	k8s.io/metrics => k8s.io/metrics v0.25.11
-	k8s.io/mount-utils => k8s.io/mount-utils v0.25.11
-	k8s.io/pod-security-admission => k8s.io/pod-security-admission v0.25.11
-	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.25.11
+	k8s.io/api => k8s.io/api v0.27.10
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.27.10
+	k8s.io/apimachinery => k8s.io/apimachinery v0.27.10
+	k8s.io/apiserver => k8s.io/apiserver v0.27.10
+	k8s.io/cli-runtime => k8s.io/cli-runtime v0.27.10
+	k8s.io/client-go => github.com/rancher/client-go v1.27.4-rancher1
+	k8s.io/cloud-provider => k8s.io/cloud-provider v0.27.10
+	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.27.10
+	k8s.io/code-generator => k8s.io/code-generator v0.27.10
+	k8s.io/component-base => k8s.io/component-base v0.27.10
+	k8s.io/component-helpers => k8s.io/component-helpers v0.27.10
+	k8s.io/controller-manager => k8s.io/controller-manager v0.27.10
+	k8s.io/cri-api => k8s.io/cri-api v0.27.10
+	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.27.10
+	k8s.io/dynamic-resource-allocation => k8s.io/dynamic-resource-allocation v0.27.10
+	k8s.io/kms => k8s.io/kms v0.27.10
+	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.27.10
+	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.27.10
+	k8s.io/kube-proxy => k8s.io/kube-proxy v0.27.10
+	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.27.10
+	k8s.io/kubectl => k8s.io/kubectl v0.27.10
+	k8s.io/kubelet => k8s.io/kubelet v0.27.10
+	k8s.io/kubernetes => k8s.io/kubernetes v1.27.10
+	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.27.10
+	k8s.io/metrics => k8s.io/metrics v0.27.10
+	k8s.io/mount-utils => k8s.io/mount-utils v0.27.10
+	k8s.io/pod-security-admission => k8s.io/pod-security-admission v0.27.10
+	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.27.10
+	oras.land/oras-go => oras.land/oras-go v1.2.2 // for docker 20.10.x compatibility
 )
 
 require (
 	github.com/gorilla/mux v1.8.0
-	github.com/rancher/norman v0.0.0-20230426211126-d3552b018687
-	github.com/rancher/rancher v0.0.0-20230712102934-01a8529371b2
+	github.com/rancher/norman v0.0.0-20240205154641-a6a6cf569608
```

Comment:
Points to https://github.com/rancher/norman/commit/a6a6cf5696088c32002953d36b75bdcc84f2399e